todo/models.py

```diff
@@ -0,0 +1,25 @@
+from django.db import models
+
+
+class Tag(models.Model):
+    name = models.CharField(max_length=100)
+
+    class Meta:
+        ordering = ["name"]
+
+    def __str__(self):
+        return f"{self.name}"
+
+
+class Task(models.Model):
+    content = models.CharField(blank=True, max_length=15)
```

Comment:
It's better to set `blank=False`, because if you set it as True, an empty string("") will be saved in the database. And if you go to `127.0.0.1:8000`, you will see the list of your tasks. The problem is that it will not be clear, which task or even tasks should be done till some day.

Would also make this(`content`) field unique, cause it's not logical to have two or more tasks with the same name, but with different deadline times.